configure.ac

```diff
@@ -99,9 +99,6 @@ if test -z "$OBJCOPY"; then
     AC_MSG_ERROR(objcopy not found)
   fi
   OBJCOPY="$LOBJCOPY"
-  O_BDFARCH="elf64-x86-64"
-else
-  O_BDFARCH=`$OBJCOPY --info | head -n 2 | tail -n 1`
```

Comment:
Why not to remove the entire objcopy
block then? Or is it still used?

Reply:
yes it's still used a little lower in
```
if test -z "$XOBJCOPY"; then
  XOBJCOPY="$OBJCOPY"
fi
```